tests/unit/modules/network/ios/test_ios_ospf_interfaces.py

```diff
@@ -40,7 +40,7 @@ def setUp(self):
         )
 
         self.mock_get_resource_connection_facts = patch(
-            "ansible_collections.ansible.netcommon.plugins.module_utils.network.common.resource_module."
+            "ansible_collections.ansible.netcommon.plugins.module_utils.network.common.resource_module_base."
```

Comment:
This will change if we go ahead with the suggestion mentioned in https://github.com/ansible-collections/ansible.netcommon/pull/180#pullrequestreview-564063342.